lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -255,13 +257,13 @@ <h3 class="error-msgs" id="error-msg"></h3>
          }
          if($('#invoice_number').val() == "") {
                      $('#error-msg').attr('class','error-msgs')
-                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.')}").show();
+                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.') | n, js_escaped_string}").show();
```

Comment:
JavaScript in Mako gets weird.  I do have a note for this.  You should use `| n, h, js_escaped_string`.  This is almost the same as `| js_escaped_string` without the `n`, but we want to make these odd cases more explicit.